http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppService.scala

```diff
@@ -14,7 +15,11 @@ trait AppService[F[_]] {
     appName: AppName,
     req: CreateAppRequest,
     workspaceId: Option[WorkspaceId] = None
-  )(implicit as: Ask[F, AppContext]): F[Unit]
+  )(implicit
+    as: Ask[F, AppContext],
+    googleResourceService: GoogleResourceService[F],
```

Comment:
It would be nice to split `AppService` into `AppService` and `AppV2Service` or something (similar to `RuntimeService` and `RuntimeV2Service`). Then we could just not instantiate `AppService` for Azure.